main.go

```diff
@@ -32,6 +34,25 @@ func main() {
 		fmt.Printf("usage: %s path\n", os.Args[0])
 		os.Exit(1)
 	}
+
+	if *template_flag {
+		templ, err := ioutil.ReadFile(flag.Args()[0])
```

Comment:
@vidriduch It may be good to print some usage here -- Some may be quite confused as to what parameters are expected.